docs/guides/collaboration/migrating-from-noodl-hosted-git.mdx

```diff
@@ -24,9 +24,23 @@ To do so, open your project in CS Noodl. Then open the Version Control panel, an
 Learn more about the Version Control panel in the [version control docs](/docs/guides/collaboration/version-control).
 :::
 
+## Move the project
+
+CS Noodl projects live in system folders that are pretty cumbersome to navigate to. A strong suggestion is to move your project to an easily accessible folder.
```

Comment:
Should `CS ` be included?

Reply:
I think it's good to have there to avoid confusion, since we talk about both the CS and OS versions in this doc. But this is probably the only place that it is needed.